src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -689,6 +833,34 @@ class BerdyHelper
                                           const FrameIndex & floatingFrame,
                                           const Vector3 & angularVel);
 
+    /**
+      * @name Methods to submit the input data for dynamics computations.
+      */
+    //@{
+
+    /**
+     * Set the kinematic information necessary for the dynamics estimation using the
+     * angular velocity information of a floating frame.
+     *
+     * \note This method cannot be used if the selected BerdyVariant is ORIGINAL_BERDY_FIXED_BASE.
+     * \note we not require to give the linear velocity of floating base because the dynamics equations
+     *       are invariant with respect to an offset in linear velocity. This convenient to avoid
+     *       any dependency on any prior floating base estimation.
+     *
+     * @param[in] baseTransform the transformation from floating base to world
```

Comment:
What do yo mean "from floating base to world" . I think using a notation like W_H_B or similar is more clear.

Reply:
Done in https://github.com/robotology/idyntree/pull/783/commits/bc56f3aeae3daab557e5ac6c946a2baf61dc979e